### PR TITLE
Add presentmon

### LINKED ISSUE #3324: [Request] PresentMon
Also making a PR where I add a manifest for this, making this just in case it's needed anyways

Edit: here: #3325

### Criteria

For a package to be acceptable in the Main bucket, it should be:

- [X] a non-GUI tool
- [X] a reasonably well-known and widely used developer tool
- [X] the latest stable version of the program
- [X] the full version i.e. not a trial version
- [X] a fairly standard install (e.g. uses a version-specific download URL, no elaborate pre/post install scripts)

### Information

**Name:** PresentMon

**Description:** Tool for collection and processing of ETW events related to frame presentation on Windows.

**Homepage:** https://github.com/GameTechDev/PresentMon

**Download link(s):**
https://github.com/GameTechDev/PresentMon/releases/download/v1.7.0/PresentMon-1.7.0-x64.exe
https://github.com/GameTechDev/PresentMon/releases/download/v1.7.0/PresentMon-1.7.0-x86.exe

**Some indication of popularity/repute:** Github stars, also used by reputable software already added in Extras bucket (CapFrameX)

### PR DESCRIPTION
**Pull request by couleurm**
Closes #3324

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
You can use a URL fragment (`url#/presentmon.exe`) to rename the file in the URL itself. No need for pre_install.

File: bucket/presentmon.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "32bit":{
+            "url": "https://github.com/GameTechDev/PresentMon/releases/download/v1.7.0/PresentMon-1.7.0-x86.exe",
+            "hash": "163999393086B349C09FDBAC74FB1C151564FFB07FF094F4E4ADCE22FD5D8098"
+        }
+    },
+    "bin": [
+        "PresentMon.exe",
+        [
+            "PresentMon.exe",
+            "pm"
```
Comment (rashil2000) [inline bucket/presentmon.json:21]: Any documentation on this particular alias?

Comment (couleurm) [reply bucket/presentmon.json:21]: no

Comment (rashil2000) [reply bucket/presentmon.json:21]: Then why add it?

Comment (couleurm) [reply bucket/presentmon.json]: to be frank I like it and tried to snug it into the Main bucket

Comment (rashil2000) [reply bucket/presentmon.json]: The next release of Scoop will add the ability to add custom aliases, you can use that then

**Comment by rashil2000** [comment]
> You can use a URL fragment (`url#/presentmon.exe`) to rename the file in the URL itself. No need for pre_install.

**Pull request merged**